.gitignore

```diff
@@ -28,13 +28,13 @@
 *.zip
 *.tar.gz
 *.rar
-*.xml
 # virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
 hs_err_pid*
 
 #idea folder
 .idea/
 out
+target
```

Comment:
add *.iml, these files are for JetBrain products =))

Reply:
Resolved 👯